protonj2/src/main/java/org/apache/qpid/protonj2/buffer/ProtonBuffer.java

```diff
@@ -1220,7 +1220,7 @@
      *            {@code src.capacity}, or
      *         if {@code length} is greater than {@code this.writableBytes}
      */
-    ProtonBuffer writeBytes(ProtonBuffer source, int sourrceIndex, int length);
+    ProtonBuffer writeBytes(ProtonBuffer source, int sourceIndex, int length);  // TODO rename to offset as in overrides?
```

Comment:
The methods in derived classes call this parameter `offset`. Maybe rename it here?

Reply:
Not a spelling error

Reply:
But do you think the rename is worth a separate Jira, or should I just fix the spelling error and move on?